test/general-tests/starknet-verify/check.sh

```diff
@@ -1,28 +1,28 @@
 #!/bin/bash
 
-echo "Temporarily disabled. Uncomment the tests once we receive API instructions from the Voyager team (Nethermind)"
 
-# set -e
+set -e
 
-# MAIN_CONTRACT=contracts/contract.cairo
-# UTIL_CONTRACT=contracts/util.cairo
+MAIN_CONTRACT=contracts/contract.cairo
+UTIL_CONTRACT=contracts/util.cairo
 
-# npx hardhat starknet-compile $MAIN_CONTRACT $UTIL_CONTRACT
+npx hardhat starknet-compile $MAIN_CONTRACT $UTIL_CONTRACT
 
-# echo "Waiting for deployment to be accepted"
-# output=$(npx hardhat starknet-deploy --starknet-network "$NETWORK" contract --inputs 10 --wait)
-# address=$(echo $output | sed -r "s/.*Contract address: (\w*).*/\1/")
+echo "Waiting for deployment to be accepted"
+output=$(npx hardhat starknet-deploy --starknet-network "$NETWORK" starknet-artifacts/$MAIN_CONTRACT --inputs 10 --wait)
```

Comment:
If users have two contracts with the same name in different directories, plugin will throw `Error in plugin Starknet: More than one file was found because the path provided is ambiguous, please specify a relative path`

Reply:
Ok, but do we have two contracts with the same name in our tests?

Reply:
Yeah, we don't have tests with the same name.